contracts/agToken/polygon/TokenPolygonUpgradeable.sol

```diff
@@ -183,6 +183,16 @@ contract TokenPolygonUpgradeable is
         emit TreasuryUpdated(_treasury);
     }
 
+    /// @dev Internal function for 'onlyGovernor' modifier
+    function checkGovernor() internal view {
```

Comment:
We use the syntax to start internal functions with an underscore: `checkGovernor` -> `_checkGovernor` usually.

Can you please update this?

Reply:
Sure!!